distribution/docs/src/main/resources/content/_metadataAttributes/NITF_Header-table.adoc

```diff
@@ -134,33 +134,29 @@ else current time
 |String
 |true
 
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSCTLH
-//|TODO
-//|security.dissemination-controls
-//|String
-//|true
-
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSCTLH
-//|TODO
-//|ext.nitf.file-control-and-handling
-//|String
-//|true
-
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSREL
-//|TODO
-//|security.releasability
-//|String
-//|true
-
-// TODO CAL-348 Document NITF to DDF Catalog Taxonomy Security Mappings
-//|NitfHeader.FSREL
-//|TODO
-//|ext.nitf.file-releasing-instructions
-//|String
-//|true
+|NitfHeader.FSCTLH
+|
+|security.dissemination-controls
+|String
+|true
+
+|NitfHeader.FSCTLH
+|
```

Comment:
Should we have `none` as the default value? Looks like the other entries in this table use that convention.